cumulus/parachains/runtimes/glutton/glutton-westend/src/lib.rs

```diff
@@ -263,7 +264,7 @@ construct_runtime! {
 		CumulusXcm: cumulus_pallet_xcm::{Pallet, Call, Storage, Event<T>, Origin} = 10,
 
 		// The main stage.
-		Glutton: pallet_glutton::{Pallet, Call, Storage, Event, Config<T>} = 20,
+		GluttonWestend: pallet_glutton::{Pallet, Call, Storage, Event, Config<T>} = 20,
```

Comment:
This can stay, it's just the pallet name. The context is already Westend.
```suggestion
		Glutton: pallet_glutton::{Pallet, Call, Storage, Event, Config<T>} = 20,
```

Reply:
Thanks, I've renamed these again